lib/vagrant-libvirt/templates/domain.xml.erb

```diff
@@ -131,7 +131,12 @@
 <% @inputs.each do |input| %>
     <input type='<%= input[:type] %>' bus='<%= input[:bus] %>'/>
 <% end %>
-
+    <% if @sound_type != 'none' %>
```

Comment:
I'd switch to a simple check of:
```
    <% if !@sound_type.nil? %>
```
And then set the default in finalize! above to `nil` when it's current `UNSET_VALUE`.

That should skip adding the sound block unless sound is configured.